index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="logo-nombre">Gustavo Porcilio</a>
+      <div class="hamburger">
+        <nav class="navlinks">
+          <a href="#porfolio">PORFOLIO</a>
+          <a href="#about">ABOUT</a>
+          <a href="#contact">CONTACT</a>
+        </nav>
+      </div>
+    </header>
+    <section class="principal">
+      <div class="profession">
+        <h1 class="tittle">Hey There! I'm Gustavo</h1>
+        <h2>I'm a software developer</h2>
+      </div>
+
+      <div class="text">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="list">
+          <li>
+            <a href="https://github.com/MarcoAurelioAntonio" class="github"></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/gustavo-porcilio-4496a223a/"
+              class="linkedin"
+            ></a>
+          </li>
+          <li><a href="#" class="angelist"></a></li>
+          <li>
+            <a href="https://twitter.com/gustavoporcilio" class="twitter"></a>
+          </li>
+          <li><a href="#" class="medium"></a></li>
+        </ul>
+      </div>
+    </section>
+    <section class="work">
+      <div class="work-tittle">
+        <h1>My Recent Work</h1>
```

Comment:
- [ ] [OPTIONAL] Please make sure you have just one `h1` tag in your HTML file as it is best practice to have just one `h1`tag in a website 🙏